integration-tests/verify_oncotree_code_converter.py

```diff
@@ -0,0 +1,46 @@
+#!/bin/python
+
+import os
+import sys
+import linecache
+
+oncotree_code_converter_output = sys.argv[1]
+if not os.path.exists(oncotree_code_converter_output):
+    print "Error, specified file not found: " + oncotree_code_converter_output
+    sys.exit(1)
+
+REQUIRED_HEADERS = ["ONCOTREE_CODE", "CANCER_TYPE", "CANCER_TYPE_DETAILED"]
+
+# returns a dictionry from column name to list of column values
+# exits if one of the columns has empty value
+def get_required_columns(oncotree_code_converter_output):
+    with open(oncotree_code_converter_output) as file:
+        header_processed = False
+        header_index_mapping = {}
+        to_return = {header : [] for header in REQUIRED_HEADERS}
+        for line in file.readlines():
+            data = line.rstrip().split("\t")
+            if not header_processed:
```

Comment:
You can just use Python's csv.DictReader, which will read the header and then you can access the values using the expected keys like you are below: https://docs.python.org/2/library/csv.html

Reply:
this is way easier!! I should have used this in all my scripts